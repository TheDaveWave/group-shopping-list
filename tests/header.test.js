import React from 'react';
// import renderer from 'react-test-renderer';
const renderer = require('react-test-renderer');
import Header from '../src/components/Header/Header';

test('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
});
