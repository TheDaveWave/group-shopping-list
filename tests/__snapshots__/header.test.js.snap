// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="banner-header"
>
  <div
    id="buyStuff"
  >
    <h1>
      BUY 
    </h1>
    <div
      id="tagHolder"
    >
      <div
        id="tag"
      >
        $
      </div>
    </div>
    <h1>
      STUFF
    </h1>
  </div>
</header>
`;
